describe/describe.ipynb

```diff
@@ -1,171 +1,41 @@
 {
```

Comment:
<p>Please run black formatter on the notebook</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mlrun/functions/pull/93/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>